### PR TITLE
Document update for Parquet Writer Versions V1 and V2 support

### PR DESCRIPTION
**Pull request by nmahadevuni**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -730,6 +730,21 @@ The following operations are not supported when ``avro_schema_url`` is set:
 * Using partitioning(``partitioned_by``) or bucketing(``bucketed_by``) columns are not supported in ``CREATE TABLE``.
 * ``ALTER TABLE`` commands modifying columns are not supported.
 
+Parquet Writer Version
+----------------------
+
+Presto now supports Parquet writer versions V1 and V2 for Hive catalog.
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/hive.rst:736]: nit: `for the Hive Catalog`.

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -730,6 +730,21 @@ The following operations are not supported when ``avro_schema_url`` is set:
...
+It can be toggled using session property ``parquet_writer_version`` and config property ``hive.parquet.writer.version``.
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/hive.rst:737]: `using the session property`
`and the config property`

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 3]
```diff
@@ -730,6 +730,21 @@ The following operations are not supported when ``avro_schema_url`` is set:
...
+Valid values for these properties are PARQUET_1_0 and PARQUET_2_0. Default is PARQUET_2_0.
+
+E.g., To set for a session, use below command by replacing <catalog-name> with actual catalog name:
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/hive.rst:740]: Is the example needed? These are regular session and config properties, right?

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 4]
```diff
@@ -682,6 +682,21 @@ schema evolution, such as adding, dropping, and renaming columns. With schema
 evolution, users can evolve a table schema with SQL after enabling the Presto
 Iceberg connector.
 
+Parquet Writer Version
+----------------------
+
+Presto now supports Parquet writer versions V1 and V2 for Iceberg catalog.
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:688]: same comments as above.

**Review by majetideepak — COMMENTED** [review]
@nmahadevuni thank you for adding the doc. A couple of minor comments.

File: presto-docs/src/main/sphinx/connector/hive.rst (hunk 3 above)
Comment (nmahadevuni) [reply presto-docs/src/main/sphinx/connector/hive.rst:740]: Yeah,  we can remove the examples

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 5]
```diff
@@ -682,6 +682,13 @@ schema evolution, such as adding, dropping, and renaming columns. With schema
 evolution, users can evolve a table schema with SQL after enabling the Presto
 Iceberg connector.
 
+Parquet Writer Version
+----------------------
+
+Presto now supports Parquet writer versions V1 and V2 for the Iceberg catalog.
+It can be toggled using the session property ``parquet_writer_version`` and the config property ``hive.parquet.writer.version``.
+Valid values for these properties are PARQUET_1_0 and PARQUET_2_0. Default is PARQUET_2_0.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:690]:
```suggestion
Valid values for these properties are ``PARQUET_1_0`` and ``PARQUET_2_0``. Default is PARQUET_2_0.
```

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 6]
```diff
@@ -730,6 +730,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 * Using partitioning(``partitioned_by``) or bucketing(``bucketed_by``) columns are not supported in ``CREATE TABLE``.
 * ``ALTER TABLE`` commands modifying columns are not supported.
 
+Parquet Writer Version
+----------------------
+
+Presto now supports Parquet writer versions V1 and V2 for the Hive catalog.
+It can be toggled using the session property ``parquet_writer_version`` and the config property ``hive.parquet.writer.version``.
+Valid values for these properties are PARQUET_1_0 and PARQUET_2_0. Default is PARQUET_2_0.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/connector/hive.rst:738]:
```suggestion
Valid values for these properties are ``PARQUET_1_0`` and ``PARQUET_2_0``. Default is PARQUET_2_0.
```

**Comment by nmahadevuni** [comment]
Thanks @majetideepak @tdcmeehan . Have addressed your comments.

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 7]
```diff
@@ -730,6 +730,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 * Using partitioning(``partitioned_by``) or bucketing(``bucketed_by``) columns are not supported in ``CREATE TABLE``.
 * ``ALTER TABLE`` commands modifying columns are not supported.
 
+Parquet Writer Version
+----------------------
+
+Presto now supports Parquet writer versions V1 and V2 for the Hive catalog.
+It can be toggled using the session property ``parquet_writer_version`` and the config property ``hive.parquet.writer.version``.
+Valid values for these properties are ``PARQUET_1_0`` and ``PARQUET_2_0``. Default is PARQUET_2_0.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/connector/hive.rst:738]: One last thing, for the below too
```suggestion
Valid values for these properties are ``PARQUET_1_0`` and ``PARQUET_2_0``. Default is ``PARQUET_2_0``.
```

**Review by tdcmeehan — CHANGES_REQUESTED** [review]

**Review by tdcmeehan — APPROVED** [review]

**Review by majetideepak — APPROVED** [review]

**Pull request merged**